sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -230,12 +242,21 @@ def _to_rest_object(self) -> ComputeResource:
                 )
             )
 
+        idle_time_before_shutdown = None
+        if self.idle_time_before_shutdown_minutes:
+            idle_time_before_shutdown = f"PT{self.idle_time_before_shutdown_minutes}M"
+        elif self.idle_time_before_shutdown:
+            module_logger.warning(
```

Comment:
Deprecation warnings are usually formatted like this:
```python
import warnings
warnings.warn(
    "This property is deprecated",
    DeprecationWarning
)
```

Reply:
I've updated this to match the format now.